soc/wch/ch32v00x/soc.yml

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2024 Michael Hope
+# SPDX-License-Identifier: Apache-2.0
+
+series:
+  - name: ch32v00x
```

Comment:
It seems to be

```diff
-name: ch32v00x
+name: ch32v
```

Reply:
I think ch32v00x is correct, as it matches the reference manual https://www.wch-ic.com/downloads/CH32V00XRM_PDF.html

Note that WCH seem to treat ch32v{2,3}xx, ch32v1xx, and ch32v0xx as separate families.